src/store/experience/actions.ts

```diff
@@ -0,0 +1,23 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import { IAddCareer } from 'src/common/types';
+import { careerApi } from 'src/services';
+import { ActionTypes } from './common';
+
+const getCareerExperience = createAsyncThunk(
```

Comment:
It's better to name async actions via load not get
```suggestion
const loadCareerExperience = createAsyncThunk(
```